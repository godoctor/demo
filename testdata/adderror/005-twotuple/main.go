package main

import "fmt"

func foo() (int, string) { //<<<<<adderror,5,1,5,1,pass
	return 5, "foo"
}

func main() {
	foo()
	fmt.Println("Done")
}
