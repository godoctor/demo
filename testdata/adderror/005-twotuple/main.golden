package main

import "fmt"

func foo() (int, string, error) { //<<<<<adderror,5,1,5,1,pass
	return 5, "foo", nil
}

func main() {
	foo()
	fmt.Println("Done")
}
